### ISSUE TITLE
Ender IO Crash with ProjectRed or StorageDrawers

### ISSUE THREAD

**Issue by EntropyMu** [Cazadorsniper/PO3#341]
Pack Version: Singleplayer, Multiplayer, or LAN?: .40 Multiplayer

Description: I use projectred pipes to push/pull items into a storage drawers storage setup.  Since .40, EnderIO items with an energy value/buffer crash when being put into said system.

Steps to reproduce:I use routed interface pipes over a chest and the drawer controller.  Using an extractor chip in the pipe over a chest, it pulls items and places them into the storage drawers.  Place an enderIO item with energy (crash has been reproduced using zombie generators from a loot chest and an enderio magnet) and the crash occurs.  

Crash log (use pastebin or similar):
https://pastebin.com/NmwNXCrb

**Comment by Cazadorsniper** [comment]
does this happen with YABBA too?

**Comment by EntropyMu** [comment]
I haven't tested that, I'll check.

**Comment by EntropyMu** [comment]
An update: With more testing, I've determined that sending the enderIO items with energy into the storage drawer system using other item piping methods doesn't cause any problems.  It must be an interaction between the projectred extractor chip/ junction pipes and the items.  Pulling items out of storage using projectred chips and the request pipe doesn't cause the crash.  I've also added the pastebin of the crash report.
